mod/thewire/views/default/resources/thewire/friends.php

```diff
@@ -2,9 +2,10 @@
 /**
  * Wire posts of your friends
  */
+$username = elgg_extract('username', $vars);
 
-$owner = elgg_get_page_owner_entity();
-if (!$owner instanceof ElggUser) {
+$owner = get_user_by_username($username);
+if (!$owner) {
```

Comment:
This is done by the magic of the page_owner, why change it?

Reply:
This is done in the same way as for blogs.
I don't see any difference.